sklearn/multiclass.py

```diff
@@ -795,6 +809,8 @@ def partial_fit(self, X, y, classes=None):
 
         if hasattr(self.estimators_[0], "n_features_in_"):
             self.n_features_in_ = self.estimators_[0].n_features_in_
+        if hasattr(self.estimators_[0], "feature_names_in_"):
+            self.feature_names_in_ = self.estimators_[0].feature_names_in_
```

Comment:
Could you please add a new test case to cover this line?

Reply:
I removed the line since this class validates `X` itself and it does not actually delegate to a base estimator.